src/metabase/sync/interface.clj

```diff
@@ -65,16 +66,15 @@
   "Schema for a given Table as provided in [[metabase.driver/describe-table-indexes]]."
   [:ref ::TableIndexMetadata])
 
-(mr/def ::TableMetadata
-  [:map
-   [:name                         ::lib.schema.common/non-blank-string]
-   [:schema                       [:maybe ::lib.schema.common/non-blank-string]]
-   [:fields                       [:set TableMetadataField]]
-   [:description {:optional true} [:maybe :string]]])
-
-(def TableMetadata
-  "Schema for the expected output of [[metabase.driver/describe-table]]."
-  [:ref ::TableMetadata])
+(mr/def ::FieldMetadataEntry
+  (-> (mr/schema ::TableMetadataField)
```

Comment:
why did you remove description? and any reason why you use these helpers instead of the normal syntax?

Reply:
Good question. We don't use the description anywhere. We get the description from `DatabaseMetadata` instead. https://github.com/metabase/metabase/pull/38828/files#diff-8eb3ce5146f34119083048007ec1c733c4c77e14b40be0aa84e583f1cc3ab6d1R10-R16

I removed the requirement to get the description from the interface as a way to show driver authors that they can  switch from `describe-table` to `describe-fields` without losing any information, since we only use the `fields` key from `describe-table`.